modules/stochastic_tools/doc/content/syntax/Mappings/index.md

```diff
@@ -0,0 +1,47 @@
+# Mapping System
```

Comment:
How about VariableMapping? I mean we are mapping between functions between two spaces, it is a general concept.

Reply:
But it doesnt map between functions & variables, or between two dissimilar meshes

If you can think of something flashier, it will look better on the NST highlight
In any case let's try to make it precise. I think "Variable" helps

Reply:
Dah VariableMapping*

Reply:
haha I was about to say nooooo not Transfer